frontend/src/views/ElementView.vue

```diff
@@ -108,7 +119,10 @@ const element = ref(null)
 
 const family = computed(() => element.value?.family?.name)
 const genre = computed(() => element.value?.genre)
-const usefulParts = computed(() => element.value?.usefulParts?.map((x) => x.name).filter((x) => !!x))
+const plantParts = computed(() => element.value?.plantParts?.map((x) => x.name).filter((x) => !!x))
+const toWatchParts = computed(() =>
+  element.value?.plantParts?.filter((x) => x.mustBeMonitored == true && !!x.name).map((x) => x.name)
```

Comment:
```suggestion
  element.value?.plantParts?.filter((x) => !!x.mustBeMonitored && !!x.name).map((x) => x.name)
```
Si on veut just savoir si la valeur est truthy on peut prepend `!!` comme on le fait pour le name juste après. Par contre si on veut savoir si c'est un vrai `true` boolean, il faudrait utiliser le triple égal (`x.mustBeMonitored === true`). Selon moi le double `!!` marche ici, mais je te laisse décider.

Reply:
oui j'étais plutôt sur du triple égal, mais je n'en ai mis que 2.
:eye: